### PR TITLE
9.0 crm sale marketing

### PR DESCRIPTION
**Pull request by SerpentCS**
Sale Marketing module - Copies the marketing details from lead to sale.

### CODE REVIEW

File: crm_sale_marketing/README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Usage
+=====
+
+* This module will be helpful to trace the Sales generated from the any Marketing source.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/111/9.0
+
+Bug Tracker
```
Comment (JordiBForgeFlow) [inline crm_sale_marketing/README.rst:38]: This section has changed. Please apply the same format as in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: crm_sale_marketing/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sales Marketing",
+    "summary": "Marketing Details of Sales",
+    'description': """
```
Comment (JordiBForgeFlow) [inline crm_sale_marketing/__openerp__.py]: Please remove the "description". It is already contained in README.rst

File: crm_sale_marketing/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+This module copies the marketing details while converting opportunity to
+quotation. This will help to know that how many Sales are generated from
+the any particular campaign.
+    """,
+    "version": "9.0.1.0.0",
+    "author": "Eficent Business and IT Consulting Services S.L., "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.eficent.com",
```
Comment (JordiBForgeFlow) [inline crm_sale_marketing/__openerp__.py]: Please replace with https://odoo-community.org/

**Comment by JordiBForgeFlow** [comment]
:+1:

**Comment by JordiBForgeFlow** [comment]
Tested on runbot, works as expected.

**Review by rafaelbn — APPROVED** [review]
Tested

**Pull request merged**